spec/validators/companies_house_name_validator_spec.rb

```diff
@@ -14,33 +14,39 @@ class NameValidatable
     subject { NameValidatable.new(name: name) }
 
     describe "Valid entry" do
+      after(:each) do
+        is_expected.to be_valid
+        expect(subject.errors[:name].size).to eq(0)
+      end
+
       context "With just normal chars" do
         let(:name) { "Joe Bloggs 123456 Ltd" }
-        it "is valid" do
-          is_expected.to be_valid
-          expect(subject.errors[:name].size).to eq(0)
-        end
+        it "is valid" do end
       end
 
       context "With just numbers" do
         let(:name) { "123456" }
-        it "is valid" do
-          is_expected.to be_valid
-          expect(subject.errors[:name].size).to eq(0)
-        end
+        it "is valid" do  end
       end
 
       context "With special but allowed chars" do
         let(:name) { "Joe @ Bloggs 123&456 * Ltd %$£" }
-        it "is valid" do
-          is_expected.to be_valid
-          expect(subject.errors[:name].size).to eq(0)
-        end
+        it "is valid" do end
+      end
+
+      context "With only a single special char" do
+        let(:name) { "@" }
+        it "is valid" do end
+      end
+
+      context "With only multiple special chara" do
+        let(:name) { "@ $$" }
+        it "is valid" do end
```

Comment:
The tests with `it "is valid" do end` aren't testing anything. If they are not needed, they should be removed.